renderer/BUILD.gn

```diff
@@ -1,22 +1,10 @@
 import("//build/config/features.gni")
 
 source_set("renderer") {
```

Comment:
leaving this empty `source_set` will break existing deps on `//brave/renderer` because it's now empty. You should remove the `//brave/renderer` dep in `//chrome/renderer` and add `//chrome/renderer` as a public dep of `//brave/renderer` so any other deps still work correctly

Reply:
Ah, thanks for pointing it out. Fixed it.